src/v2i-hub/TelematicBridgePlugin/manifest.json

```diff
@@ -11,6 +11,31 @@
             "key": "LogLevel",
             "default": "INFO",
             "description": "The log level for this plugin"
+        },
+        {
+            "key": "NATSUrl",
+            "default": "nats://127.0.0.1:4222",
+            "description": "The log level for this plugin"            
+        },
+        {
+            "key": "UnitId",
+            "default": "v2xhub_id",
+            "description": "The log level for this plugin"            
+        },
+        {
+            "key": "UnitName",
+            "default": "East Intersection",
+            "description": "The log level for this plugin"            
+        },
+        {
+            "key": "UnitType",
+            "default": "Infrastructure",
+            "description": "The log level for this plugin"            
+        },
+        {
+            "key": "TopicExclusionList",
```

Comment:
These are not topics but messages

Reply:
The topic name consist of type_subtype_source. We create a topic for each message because v2xhub does not have a topic.

Reply:
So topics is not a familiar term for users of V2X-HUb. If we are making a connection between topics and message in telematics, I would recommend we use "message" to refer to the messages we are listening to in V2X-Hub and "topic" as the term of the topics we will create on telematics.

I suggest we call this messages here and the term topics is only used to send information to telematics about topics to create and in the telematics UI/Software.

Reply:
Addressed https://github.com/usdot-fhwa-OPS/V2X-Hub/pull/567/commits/71ac033b94eec7ea112cb5ead2d680a29b769772